Tevling/Services/ActivityService.cs

```diff
@@ -134,7 +134,7 @@ public async Task<Activity[]> GetActivitiesAsync(
         Activity[] activities = await dataContext.Activities
             .Include(a => a.Athlete)
             .ThenInclude(a => a!.Following)
-            .Where(a => filter.Months == 0 || a.Details.StartDate >= now.AddMonths(-filter.Months))
+            .Where(a => !filter.From.HasValue|| a.Details.StartDate >= filter.From)
```

Comment:
Tiny nit, but you could use the `If` extension here:
```suggestion
            .If(filter.From.HasValue, q => q.Where(a => a.Details.StartDate >= filter.From))
```
Might improve readability, I dunno..

Reply:
I agree that it would improve the readability. 

Though, changing it as you suggested gives the error:
`Cannot convert expression type 'System.Linq.IQueryable<Tevling.Model.Activity>' to return type 'Microsoft.EntityFrameworkCore.Query.IIncludableQueryable<Tevling.Model.Activity,System.Collections.Generic.ICollection<Tevling.Model.Athlete>>`

Seems to work if I add `.asQueryabel()` before the `.If()`. Or move the `If()` further "down" the pipeline. 🤔

Reply:
Fixed: https://github.com/veloek/tevling/pull/42/commits/84183df72c7e85aa21a45c4b77acf39e665c26cd